Cython/Utility/ModuleSetupCode.c

```diff
@@ -109,6 +109,8 @@
   #define CYTHON_USE_DICT_VERSIONS 0
   #undef CYTHON_USE_EXC_INFO_STACK
   #define CYTHON_USE_EXC_INFO_STACK 0
+  #define CYTHON_UPDATE_DESCRIPTOR_DOC 1
```

Comment:
Could you please guard these with `#ifndef`s? You never know if users may have reasons to disable the flag in the future. However unlikely that is, the flag is there, so why not make it easy to configure it.

Reply:
done